src/ape/api/base.py

```diff
@@ -1,14 +1,18 @@
-from abc import ABCMeta, abstractmethod
+from abc import ABCMeta
+from abc import abstractmethod as _abstractmethod
 from functools import partial
+from typing import Callable
 
-from dataclassy import dataclass
+from dataclassy import dataclass as _dataclass
 from dataclassy.dataclass import DataClassMeta
 
 
 class AbstractDataClassMeta(DataClassMeta, ABCMeta):
     pass
 
 
+abstractmethod: Callable = _abstractmethod
```

Comment:
Ugh.

So two things here:

1. Cannot document this unless we do this
2. The docs gives annoying warnings everywhere this is used when we don't do it like this.

Reply:
rekt

Reply:
We no longer have to do this, thanks to `myst-parser`!